core/controllers/resources_test.py

```diff
@@ -242,6 +244,7 @@ class AssetDevHandlerAudioTest(test_utils.GenericTestBase):
 
     TEST_AUDIO_FILE_MP3 = 'cafe.mp3'
     TEST_AUDIO_FILE_FLAC = 'cafe.flac'
```

Comment:
Doesn't look like this is used; if so, it's fine to delete it and the corresponding file.

Reply:
Done